src/main/kotlin/org/rust/cargo/runconfig/CargoTestRunState.kt

```diff
@@ -60,7 +66,24 @@ class CargoTestRunState(
 
             addFormatJsonOption(post, "--format")
 
+            if (checkShowOutputSupport(rustcVer)) {
+                post.add("--show-output")
+            }
+
             return if (post.isEmpty()) pre else pre + "--" + post
         }
+
+        private fun checkShowOutputSupport(ver: RustcVersion?): Boolean {
+            if (ver == null) return false
+            // --show-output is supported since 1.39.0-nightly/dev with a build date later than 2019-08-27
```

Comment:
Can we check that this flag is supported without version checking? Like we do with some other flags

Reply:
I think no. The `--show-output` is only mentioned in `cargo test -- --help` output, but this command may trigger compilation.

Reply:
i don't think it's possible to detect this without compiling tests as it's a feature in libtest that's only in the resulting test binary